src/core/chp.jl

```diff
@@ -34,7 +34,7 @@ prime_movers = ["recip_engine", "micro_turbine", "combustion_turbine", "fuel_cel
 `CHP` is an optional REopt input with the following keys and default values:
 ```julia
     prime_mover::String = ""
-    fuel_cost_per_mmbtu::Union{<:Real, AbstractVector{<:Real}} = []  # REQUIRED
+    fuel_cost_per_mmbtu::Union{<:Real, AbstractVector{<:Real}} = []  # REQUIRED. Can be a scalar, a list of 12 monthly values, or a time series of values for every time step
```

Comment:
@Bill-Becker  or @zolanaj I added this comment (copied from ExistingBoiler). Is this true for CHP?

Reply:
@adfarth yes it's true, the type `Union{<:Real, AbstractVector{<:Real}}` matches the prose from your new note, and both technologies use the same function 
`per_hour_value_to_time_series` for conversion from any of these types to the model inputs.

Reply:
thanks for confirming!